docker-bake.hcl

```diff
@@ -56,6 +88,14 @@ target "misp" {
     "MISP_TAG": "${MISP_TAG}",
     "MISP_COMMIT": "${MISP_COMMIT}",
     "PHP_VER": "${PHP_VER}",
+    "PYPI_REDIS_VERSION": "${PYPI_REDIS_VERSION}",
+    "PYPI_LIEF_VERSION": "${PYPI_LIEF_VERSION}",
+    "PYPI_PYDEEP2_VERSION": "${PYPI_PYDEEP2_VERSION}",
+    "PYPI_PYTHON_MAGIC_VERSION": "${PYPI_PYTHON_MAGIC_VERSION}",
+    "PYPI_MISP_LIB_STIX2_VERSION": "${PYPI_MISP_LIB_STIX2_VERSION}",
+    "PYPI_MAEC_VERSION": "${PYPI_MAEC_VERSION}",
+    "PYPI_MIXBOX_VERSION": "${PYPI_MIXBOX_VERSION}",
+    "PYPI_CYBOX_VERSION": "${PYPI_CYBOX_VERSION}"
```

Comment:
trailing comma here missing (better to have trailing commas so diffs are easier)

Reply:
added